holoviews/operation/flatten.py

```diff
@@ -0,0 +1,19 @@
+try:
+    from .datashader import shade
+except ImportError:
+    shade = None
+
+
+class flatten_stack(shade):
```

Comment:
Since this is really just a plotting helper I'd probably move this into `plotting/util.py` Also please inline the datashader import, we should not import datashader unless the user uses this feature.